### PR TITLE
Responsive header for install template

### PR DESCRIPTION
**Pull request by chmst**
See title.
On large sceens, the joomla logo is displayed, the site title has a icon cogs.
On small screens, a small joomla replaces the other icons.
The Title is always visible in full length, the version is always visible.

### CODE REVIEW

**Comment by brianteeman** [comment]
Is there some css missing? This doesn't look right to me.

**Comment by brianteeman** [comment]
very weird maybe its my pc as it looks ok in firefox mobile emulation but not in chrome mobile emulation where it is just shrunk

**Comment by chmst** [comment]
Thank you @nadjak77

File: installation/template/index.php [hunk 1]
```diff
@@ -53,45 +53,37 @@
...
 			<?php // Header ?>
 			<header id="header" class="header">
-				<div class="d-flex align-items-center">
-					<div class="header-title d-flex mr-auto">
-						<div class="d-flex">
-							<a class="logo" href="<?php echo Route::_('index.php'); ?>">
-								<img src="<?php echo $this->baseurl; ?>/template/images/logo-joomla-blue.svg" alt=""/>
-							</a>
+				<div class="d-flex">
+					<div class="w-100 d-flex   mb-xs-3 ">
```
Comment (brianteeman) [inline installation/template/index.php]:
```suggestion
					<div class="w-100 d-flex mb-xs-3 ">
```

File: installation/template/index.php [hunk 2]
```diff
@@ -53,45 +53,37 @@
 	<head>
 		<jdoc:include type="metas" />
 		<jdoc:include type="styles" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
Comment (brianteeman) [inline installation/template/index.php]: Do we really need shrink to fit? From memory this was just to fix a bug in iOS 9.2 and nothing else

Comment (fancyFranci) [reply installation/template/index.php]: No, a quick web search says the same. We can remove shrink-to-fit=no.

Comment (brianteeman) [reply installation/template/index.php]: Thanks.

Comment (chmst) [reply installation/template/index.php]: Thank you

**Comment by brianteeman** [comment]
You are correct that we need this in the installation template but this will put the info after the other meta and style in the head. It needs to be right at the top - see https://htmlhead.dev/

If you look at all the other times we insert this code we do it as follows

https://github.com/joomla/backend-template/blob/bc08daf10b4dba01c169c4da25707bfcb89f6795/installation/template/error.php#L31

and this comes before the jdoc:include

**Review by Hackwar — APPROVED** [review]

**Pull request merged**